app/frontend/users/UserEdit.res

```diff
@@ -474,6 +498,36 @@ let make = (
                     </div>}
               </div>
             </div>
+            {state.newEmail == email
+              ? ""->str
+              : state.newEmail !== ""
+              ? <div className="mt-6">
+                <label
+                  name="user_new_email"
+                  className="block text-sm font-semibold"
+                  htmlFor="user_new_email">
+                  {t("new_email_label")->str}
+                </label>
+                <div className="mt-2 flex items-stretch gap-2">
+                  <input
+                    name="user_new_email"
+                    id="user-update__new-email-input"
+                    type_="email"
+                    ariaLabel="User New Email"
+                    disabled=true
+                    value=state.newEmail
+                    className="appearance-none mb-2 block text-sm w-full shadow-sm border border-gray-300 rounded px-4 py-2 my-2 leading-relaxed focus:outline-none focus:border-transparent focus:ring-2 focus:ring-focusColor-500"
+                    placeholder="New Email"
+                  />
+                  {state.disableEmailInput
+                    ? <button className="btn btn-secondary" onClick={_ => cancelNewEmailRequest()}>
+                        {t("cancel_new_email")->str}
+                      </button>
+                    : <button />}
+                </div>
+              </div>
+              : ""->str}
+            <div />
```

Comment:
This state in the UI is pretty confusing when you have an active email change request:

![image](https://github.com/pupilfirst/pupilfirst/assets/24313597/19e5605b-dc81-4f06-8556-034725b90ad8)

If we have an active email update request, let's have a text mentioning the same with a cancel button to revoke that request.

_You currently have an active email update request from xyz@example.com to xyz1@example.com. Please check your inbox and confirm the action. Use the Cancel button to revoke the request and retain the old email_